lunes_cms/help/templates/public_upload.html

```diff
@@ -65,28 +65,46 @@
         3: "das",
         4: "die (Plural)"
       }
+      let buffer_array = []
 
-      $( document ).ready(function() {
+      function Comparator(a, b) {
+        if (a[1] < b[1]) return -1;
+        if (a[1] > b[1]) return 1;
+        return 0;
+      }
 
+      $( document ).ready(function() {
+        $("#inputDiscipline").append(new Option("Alle Berufe", 0))
         lunes_disciplines.forEach(function(element) {
           if( element[1] !== "Allgemeine Grundlagen") {
             $("#inputDiscipline").append(new Option(element[1], element[0]))
           }
         });
-
         $("#inputDiscipline").change(function() {
           var discipline_id = $("#inputDiscipline").val();
```

Comment:
```suggestion
          let discipline_id = $("#inputDiscipline").val();
```
I'd use `let` here as well.